src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -87,9 +87,10 @@ const myAppConfig = {
 
 Amplify.configure(myAppConfig);
 ```
+[question]: # (RENE, Minor detail: In line 84 above, should "ID" be all caps in both instances?)
 
```

Comment:
Q from Jim - Minor detail: In line 79 above, should "ID" be all caps in both instances?